### PR TITLE
Add copy to image functionality

### PR DESCRIPTION
**Pull request by jmuharsky**
Chart widgets now contain a "camera" button that can be used to copy an image of the chart.  The dimensions of the chart will be the same as they are currently rendered, so you can maximize a chart then copy it to get a larger version.

### CODE REVIEW

**Comment by jmuharsky** [comment]
This code is deployed at https://28-1-dot-perfkit-explorer.appspot.com

**Comment by jmuharsky** [comment]
Note that this functionality is only supported for **core charts** and the **Geo** chart.  Support for additional charts will be added as the gviz team makes functionality available.

File: client/components/dashboard/dashboard-directive.html [hunk 1]
```diff
@@ -32,6 +32,11 @@
             <span class="glyphicon glyphicon-refresh pk-widget-button"
                   ng-show="widget.model.type === widgetFactorySvc.widgetTypes.CHART"
                   ng-click="clickRefreshWidget($event, widget)"></span>
+            <span class="fa fa-camera pk-widget-button"
+                  ng-show="widgetSvc.isCopyableAsImage(widget)"
+                  ng-click="clickCopyAsImage($event, widget)">
+              <md-tooltip>Copy the current chart as a PNG, which you can paste into your favorite program.</md-tooltip>
```
Comment (klausw) [inline client/components/dashboard/dashboard-directive.html]: How about just "Copy chart image to clipboard"? The current one is a bit confusing, the PNG itself isn't really visible anywhere and seems like an implementation detail.

File: client/components/widget/widget-service.js [hunk 2]
```diff
@@ -63,7 +65,52 @@ explorer.components.widget.WidgetService = class {
...
+   * Copies an image of the current chart to the clipboard.
+   * 
+   * @param {!WidgetConfig} widget
+   */
+  copyAsImage(widget) {
+    let chartWrapper = widget.state().chart.element;
+    let chartObject = chartWrapper.getChart();
+
+    let imageBuffer = document.getElementById('pk-chart-image-buffer');
+    let sel = window.getSelection();
```
Comment (klausw) [inline client/components/widget/widget-service.js]: Out of compat paranoia, should this line go into the try {} block also?

**Comment by klausw** [comment]
LGTM apart from minor bikeshedding.

File: client/components/dashboard/dashboard-directive.html (hunk 1 above)
Comment (jmuharsky) [inline client/components/dashboard/dashboard-directive.html]: That's much more succinct text. :)  I will change the tooltip.

File: client/components/widget/widget-service.js (hunk 2 above)
Comment (jmuharsky) [inline client/components/widget/widget-service.js]: Wouldn't hurt.  Will make the change.

**Pull request merged**